cwd/src/driver/svg/graphic/buttonTextDriver.js

```diff
@@ -0,0 +1,30 @@
+export const buttonTextDriver = () => {
+  return {
+    create: function(state) {
+      var newDiv = document.createElement("div");
```

Comment:
Please try to use `const` as much as possible.  If it cannot be avoided, please try to use `let` rather than `var`.  Scope is *so weird* with `var`.